examples/aws/cassandra/example.tfvars

```
@@ -24,6 +24,7 @@ cassy = {
   #resource_count            = "1"
   #resource_root_volume_size = "64"
   #enable_tdagent            = "true"
+  #s3_bucket_name            = ""
```

I think we can use Cassy naming for this since it's a cassy configuration and the name is abstracted.
```suggestion
  #storage_base_uri            = ""
```
And users would specify `s3://bucket-name` for AWS.

---

Good point. But we need an S3 bucket name to create a policy for EC2 instances to interact with S3. This is very AWS specific. Even if we abstract the variable name at this stage, we still need to assume that the value is an `s3://` URI since the bucket name must be extracted from it.
On the other hand [the `universal` module receives an abstracted URI](https://github.com/scalar-labs/scalar-terraform/pull/103/files#diff-84f2d0aebbe5577ab4dc44a45d3a5318R52) like `s3://bucket-name` from this `aws` module.

---

Considering similar interfaces in AWS and Azure,  `storage_base_uri` might be good.
Maybe we can use `replace` function to get bucket name. (e.g. replace(storage_base_uri, "s3://", ""))